backend/controller/controller.go

```diff
@@ -173,7 +173,10 @@ func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	route := routes[rand.Intn(len(routes))] //nolint:gosec
+
+	pathParams := getPathParams(route.Path, r.URL.Path)
+	logger.Infof("Path params: %v", pathParams)
```

Comment:
Not sure where best to put these so that they can be used/retrieved from the `verb` code. I could add it to the `ctx` and have a `PathParamsWithContext(ctx)` or something.